src/rules/completedDocsRule.ts

```diff
@@ -366,43 +366,66 @@ function walk(context: Lint.WalkContext<ExclusionsMap>, typeChecker: ts.TypeChec
             case ts.SyntaxKind.GetAccessor:
             case ts.SyntaxKind.SetAccessor:
                 if (node.parent!.kind !== ts.SyntaxKind.ObjectLiteralExpression) {
-                    checkNode(node as ts.AccessorDeclaration, ARGUMENT_PROPERTIES);
+                    checkAccessorNode(node as ts.AccessorDeclaration);
                 }
         }
 
         return ts.forEachChild(node, cb);
     }
 
     function checkNode(node: ts.NamedDeclaration, nodeType: DocType, requirementNode: ts.Node = node): void {
+        if (!nodeIsExcluded(node, nodeType, requirementNode) && !nodeHasDocs(node)) {
+            addDocumentationFailure(node, describeNode(nodeType), requirementNode);
+        }
+    }
+
+    function checkAccessorNode(node: ts.AccessorDeclaration): void {
+        if (nodeIsExcluded(node, ARGUMENT_PROPERTIES, node) || nodeHasDocs(node)) {
+            return;
+        }
+
+        const parent = node.parent as ts.ClassDeclaration | ts.ClassExpression;
+        const correspondingKind = node.kind === ts.SyntaxKind.GetAccessor
+            ? ts.SyntaxKind.SetAccessor
+            : ts.SyntaxKind.GetAccessor;
+        const correspondingAccessor = getNodeOfKind(parent.members, correspondingKind);
```

Comment:
This will return the first accessor of the class, not the corresponding accessor.
You also need to check the name. Note that typescript doesn't merge documentation of accessors with computed names. 
You can use `getPropertyName` to get the name as string or `undefined` if it's computed.